torch/testing/_internal/common_utils.py

```diff
@@ -920,9 +928,19 @@ def read_and_process():
 
     if disabled_test_from_issues is not None:
         if test_name in disabled_test_from_issues:
-            raise unittest.SkipTest(
-                "Test is disabled because an issue exists disabling it: {}".format(disabled_test_from_issues[test_name]) +
-                " To enable set the environment variable PYTORCH_RUN_DISABLED_TESTS=1")
+            issue_url, platforms = disabled_test_from_issues[test_name]
+            platform_to_conditional: Dict = {
+                "mac": IS_MACOS,
+                "macos": IS_MACOS,
+                "windows": IS_WINDOWS,
+                "linux": IS_LINUX
+            }
+            if platforms == [] or any([platform_to_conditional[platform] for platform in platforms]):
```

Comment:
the first part of this condition is redundant, right?
```suggestion
            if any([platform_to_conditional[platform] for platform in platforms]):
```

Reply:
No, cuz `any([])` would be false, no?